google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/file/DockerFileLanguage.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.file;
+
+import com.intellij.lang.Language;
+
+/**
+ * Barebones implementation for the Docker language required by {@link DockerFileType}.
+ */
+public class DockerFileLanguage extends Language {
```

Comment:
I think it'd be nice to also explicitly call these classes DockerFileTemplateLanguage and DockerFileTemplateType